acme/acme/magic_typing_test.py

```diff
@@ -0,0 +1,38 @@
+"""Tests for acme.magic_typing."""
+import mock
```

Comment:
super nit: `mock` is a 3rd party import while `sys` and `unittest` are in the standard library. Can we reorder things here to reflect this? https://www.python.org/dev/peps/pep-0008/#imports

Reply:
`$ bradlint`